scripts/transfer-doc-files.js

```diff
@@ -2,42 +2,107 @@ const path = require('path');
 const fs = require('fs');
 const { execSync } = require('child_process');
 
-const DOC_FILES_DIRECTORY = path.join(__dirname, '../docs/static/components')
+const DOC_FILES_DIRECTORY = path.join(__dirname, '../docs/static/components');
+const DOC_FILES_CONTENT_DIRECTORY = path.join(__dirname, '../docs/content/components');
+
+const frontMatterTemplate = (componentName) => {
+    return `---
+title: "${componentName.charAt(0).toUpperCase() + componentName.slice(1)}"
+date: 2020-10-08T14:00:45Z
+draft: false
+---`;
+};
 
 /**
  * Get all folder names within the components folder.
  * @returns {Array<string>}
 */
-function getComponentDirectories() {
-    return fs.readdirSync(path.join(__dirname, '../components'), { withFileTypes: false });
+function getFolderDirectories(folder) {
+    return fs.readdirSync(path.join(__dirname, folder), { withFileTypes: false });
+}
+
+/**
+ * Copies all .css files withing a folder and its nested folders
+ * Same names are ignored, will be overwritten - this case is so
+ * uncommon that it is easier to leave it for manual fix, instead of
+ * doing it programmatically
+ *
+ * @param {string} folder - the current folder that will be searched for css files
+ * @param {string} component - the name of the current component
+*/
+function copyAllStyleFiles(folder, component) {
+    const folderFiles = fs.readdirSync(folder);
+
+    for (let file of folderFiles) {
+        if (file === 'components-theme.css' || file === 'demo') continue;

```

Comment:
**gameface-**components-theme.css?

Reply:
`'demo'` without file extension?

Reply:
Maybe it should be checked with indexOf('demo') !== -1 because now the router has two demo.html files :D

Reply:
What is more IMO you can use Set here - `const EXCLUDED_FILES = new Set(['components-theme.css', 'demo.html'])` and then
```
for (let file of folderFiles) {
        if (EXCLUDED_FILES.has(file)) continue;
```
if you will compare with `===` :D because like that it is more readable.

Reply:
1. - Yes, it must be gameface-components-theme.css
2. - We check for the demo/ folder, not the file, this is why it is without extension
3. We don't copy the html files, because they are not exactly the same.

Reply:
Added the change with the Set.